src/scripts/party/PartyPokemon.ts

```diff
@@ -18,21 +19,25 @@ class PartyPokemon implements Saveable {
         attackBonusPercent: 0,
         attackBonusAmount: 0,
         proteinsUsed: 0,
+        effortPoints: 0,
```

Comment:
This is not used for anything, right? EP are saved in statistics. There is no reason for us to keep track of the same value twice.